app/build.gradle

```diff
@@ -27,8 +30,15 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.3.1'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile('com.crashlytics.sdk.android:crashlytics:2.6.8@aar') { transitive = true; }
```

Comment:
You added the Crashlytics, but I didn't see it being registered in the Application class. Remember to start it just in  the release build type.